pandas-stubs/core/indexing.pyi

```diff
@@ -56,8 +56,14 @@ class _LocIndexerFrame:
     @overload
     def __getitem__(self, idx: _MaskType,) -> DataFrame: ...
     @overload
+    def __getitem__(self, idx: List[int],) -> DataFrame: ...
```

Comment:
is this for dataframes with numeric indices? can you not use `.iloc` for this?

Reply:
According to the tests, you're absolutely right